dashboard/client/src/components/ActorTable.tsx

```diff
@@ -59,8 +98,19 @@ const ActorTable = ({
   const [actorIdFilterValue, setActorIdFilterValue] = useState(filterToActorId);
   const [pageSize, setPageSize] = useState(10);
   const { ipLogMap } = useContext(GlobalContext);
+
   const actorList = Object.values(actors || {}).filter(filterFunc);
-  const list = actorList.slice((pageNo - 1) * pageSize, pageNo * pageSize);
+
+  const sortActorsCallback = useCallback(() => {
+    const sortedActors = sortActors(actorList);
+    setSortedActors(sortedActors);
+  }, [actorList]);
+
+  useEffect(() => {
+    sortActorsCallback();
+  }, [sortActorsCallback]);
```

Comment:
Instead of doing this as a useEffect, can we create sortedActor via a `useMemo`? I think that will simplify the code alot. We can get rid of the sortedActor state and the sortActorsCallback.